archive/components/reports/ocpCloudReportSummary/ocpCloudReportSummaryDetails.tsx

```diff
@@ -41,7 +41,6 @@ const OcpCloudReportSummaryDetailsBase: React.SFC<OcpCloudReportSummaryDetailsPr
 }) => {
   let cost: string | React.ReactNode = <EmptyValueState />;
   let infrastructureCost: string | React.ReactNode = <EmptyValueState />;
-  let markupCost: string | React.ReactNode = <EmptyValueState />;
```

Comment:
Files that are under the archive directory are obsolete and meant to be reference.

Reply:
Sure. I can put the files back but the git repo commits keep the history, so you could just delete this folder then and refer to your SCM.